packages/devtools-reps/src/launchpad/components/Result.js

```diff
@@ -83,6 +83,9 @@ class Result extends Component {
         createLongStringClient,
         releaseActor,
         mode: MODE[modeKey],
+        onCmdCtrlClick: (node, { depth, event, focused, expanded }) => {
+          console.log("CmdCtrlClick", {node, depth, event, focused, expanded});
```

Comment:
Could you add a comment here explaining that the console.log is for verifying that this functionality works in launchpad? Otherwise I think it's a bit confusing why it's there.